### PR TITLE
LLM micro service input data does not have input model name

### PR DESCRIPTION
**Pull request by xiguiw**
vLLM and oLLAMa get the model name from the envrionment

### CODE REVIEW

**Review by lvliang-intel — APPROVED** [review]

**Comment by Spycsh** [comment]
@xiguiw why OLLAMA needs this? https://github.com/xiguiw/GenAIComps/blob/main/comps/cores/proto/docarray.py#L81 here has the input model name instead of using another env variable

File: comps/llms/text-generation/ollama/llm.py [hunk 1]
```diff
@@ -49,4 +49,5 @@ async def stream_generator():
 
 if __name__ == "__main__":
     ollama_endpoint = os.getenv("OLLAMA_ENDPOINT", "http://localhost:11434")
+    model_name = os.getenv("OLLAMA_MODEL", "meta-llama/Meta-Llama-3-8B-Instruct")
```
Comment (Spycsh) [inline comps/llms/text-generation/ollama/llm.py:52]: Why meta-llama/Meta-Llama-3-8B-Instruct?

Comment (xiguiw) [reply comps/llms/text-generation/ollama/llm.py:52]: It's default value. In case OLLAMA_MODEL is not set, then it is 'meta-llama/Meta-Llama-3-8B-Instruct'

Comment (Spycsh) [reply comps/llms/text-generation/ollama/llm.py:52]: model=input.model if input.model else model_name

**Pull request merged**